operator/v1/types_etcd.go

```diff
@@ -43,6 +43,17 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// backendQuotaGiB sets the etcd backend storage size limit in gibibytes.
+	// The value should be an integer not less than 8 and not more than 32.
+	// When not specified, the default value is 8.
+	// +kubebuilder:validation:Minimum=8
```

Comment:
@JoelSpeed adding `minimum` and removing `default` is the only way the `integration` does not fail